conf/map/battle/skill.conf

```diff
@@ -324,3 +324,9 @@ bowling_bash_area: 0
 // punch a hole into SG it will for example create a "suck in" effect.
 // If you disable this setting, the knockback direction will be completely random (eAthena style).
 stormgust_knockback: true
+
+// In eAthena, Magic Rod's animation would not occur unless a spell was absorbed.
+// In official, Magic Rod's animation occurs every time it is used.
+// false: Official behavior
+// true: Old eAthena behavior
+eathena_magicrod: false
```

Comment:
// Magic Rod's animation behavior (Note 1)
// 0 : (official) Magic Rod's animation occurs every time it is used.
// 1 : Magic Rod's animation would not occur unless a spell was absorbed. (old behavior)
+eathena_magicrod: false

though ... honestly I don't know about this kind of PR

Reply:
I think it's better configuration name is

magicrod_type: 0

Reply:
this work great!.. i did change mine into oldbehavior_magicrod